src/Conversions.ts

```diff
@@ -0,0 +1,18 @@
+import { toBN, fromWei, toWei } from 'web3-utils';
+/**
+ * TODO: Hard-coded values: for testing purposes only!
+ */
+ export function getWeiFromRifWei(trifWei: BN): BN {
```

Comment:
is `getRBTCWeiFromRifWei` too ugly? it is, isn't it? :joy:

Reply:
No, it isn't ;-)